plugins/kubernetes/app/models/kubernetes/api/pod.rb

```diff
@@ -59,16 +49,8 @@ def reason
         reasons.reject(&:blank?).uniq.join("/").presence || "Unknown"
       end
 
-      def deploy_group_id
-        Integer(labels.fetch(:deploy_group_id))
-      end
-
-      def role_id
-        Integer(labels.fetch(:role_id))
-      end
-
-      def containers
-        @pod.dig(:spec, :containers)
+      def container_names
+        (@pod.dig(:spec, :containers) + self.class.init_containers(@pod)).map { |c| c.fetch(:name) }.uniq
```

Comment:
why is init_containers not a method on the instance?

Reply:
used from outside too